### DISCUSSION TITLE
[German] Speicherort der Einstellungen

### DISCUSSION THREAD

**Discussion by KingStylesDE** [Komet/MediaElch#1726] [Q&A] [answered]
Hallo,

jedes mal wenn ich Windows neu installiere muss ich sämtliche Einstellungen in MediaElch neu einstellen (z.B. Pfade der Media Ordner usw.) obwohl ich den Ordner kvibes im Local Ordner sichere. Wo liegen diese Einstellungen damit ich diese sichern kann?

**Comment by bugwelle** [comment, marked as answer]
Hi,

unter Windows werden die Einstellungen in der Windows Registry gespeichert.
Der Registry Pfad lautet `HKEY_CURRENT_USER/Software/kvibes/MediaElch`.

Da ich selbst kein Windows nutze, kann ich leider nicht dabei helfen, die Einstellungen dort zu exportieren/importieren.
Ein "Einstellungen Exportieren"-Feature fehlt in MediaElch.

Dokumentation auf Englisch: https://mediaelch.github.io/mediaelch-doc/faq.html#where-are-mediaelch-s-settings-stored

Grüße
Andre

**Reply by KingStylesDE** [reply]
Perfekt! Vielen Dank! Import und Export ist kein Problem. Ja das Feature fehlt wirklich
